bpm/bonita-core/bonita-process-instance/bonita-process-instance-impl/src/main/resources/org/bonitasoft/engine/core/process/instance/model/impl/hibernate/process.instance.queries.hbm.xml

```diff
@@ -83,6 +83,7 @@
 		WHERE g.logicalGroup4 = :parentProcessInstanceId
 		AND g.name = :name
 		AND g.terminal = FALSE
+		AND g.hitBys NOT LIKE 'FINISH%'
```

Comment:
I did some profiling and did not notice any change on this query (but with few data in the database)

Some explain on postgres on this query give me this:
* without not like:
```Seq Scan on flownode_instance sgatewayin0_  (cost=0.00..2.02 rows=1 width=3350)
  Filter: ((NOT terminal) AND (1 = tenantid) AND ((kind)::text = 'gate'::text) AND (logicalgroup4 = 18) AND ((name)::text = 'gateway1'::text))```
* with not like:
```Seq Scan on flownode_instance sgatewayin0_  (cost=0.00..2.02 rows=1 width=3350)
  Filter: ((NOT terminal) AND ((hitbys)::text !~~ 'FINISH%'::text) AND (1 = tenantid) AND ((kind)::text = 'gate'::text) AND (logicalgroup4 = 18) AND ((name)::text = 'gateway1'::text))```


Do you think it's ok to keep it like that or should we still change the way to retrieve unfinished gateway (adding a flag and so on)